### DISCUSSION TITLE
GitHub API

### DISCUSSION THREAD

**Discussion by chakradhar-kadiyala-telus** [community/community#49429] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Hi,
I am trying to use Github API to trigger a workflow from external sources using HTTP curl command. However, I also need to know the status of the job from the same script through API only. I could trigger a workflow. But wanted to know if it's possible to know the status

**Comment by ettaboyle** [comment]
Hi @chakradhar-kadiyala-telus, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails:
- **updating the title** 
- giving a few more details might help someone give you a nudge in the right direction.

Cheers!

**Reply by chakradhar-kadiyala-telus** [reply]
Thanks.
